### ISSUE TITLE
yhat_upper and yhat_lower

### ISSUE THREAD

**Issue by Man1978-scd** [ourownstory/neural_prophet#688]
How to calculate yhat_upper and yhat_lower like prophet, divide the data set into training set and validation set, and then use the validation error to calculate the confidence interval of the upper and lower bounds，but the parameter n_forecasts is generally not set too large, which results in insufficient evaluation samples,Is there a better way?

**Comment by Kevin-Chen0** [comment]
Hi @Man1978-scd, I recently pushed the Quantile Regression (QR) feature into NeuralProphet, which allows the given model to output predictions based on inputted quantiles, say [0.05, 0.5, 0.95], where 0.5 is the mean value. This does not require using an additional data set (i.e., validation set) to calculate the intervals. [Here](https://github.com/ourownstory/neural_prophet/blob/main/tutorials/feature-use/uncertainty_estimation_peyton_manning.ipynb) is an example notebook of how to apply QR. I hope that this is what you are looking for.
